03_letters/pfannerj/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var mainout io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintf(mainout, strings.Join(sortLetters(letters("aba")), "\n"))
+
+}
+
+func letters(s string) map[rune]int {
+
+	mapOfRunes := map[rune]int{}
+	allRunes := []rune(s)
+	count := 0
+	for i := range allRunes {
+		count = mapOfRunes[allRunes[i]]
```

Comment:
I believe you can improve this with sth like:
`mapOfRunes[allRunes[i]]++`

Reply:
Yes, thx!  I have done that change and I also further simplified because I realised after reviewing someone else's solution that I don't need to create the allRunes slice at all.. I just have this now...
`	for _, thisRune := range s {
		mapOfRunes[thisRune]++
	}`